src/routes/_index/menu-mon-compte.svelte

```diff
@@ -36,6 +37,17 @@
     Mes favoris
   </a>
 
+  <a href="/alertes" class={aClass}>
+    <span
+      class="mr-s10 inline-block h-s24 w-s24 fill-current"
+      class:text-magenta-cta={$page.url.pathname === "/alertes"}
```

Comment:
```suggestion
      class:text-magenta-cta={$page.url.pathname === "/mes-alertes"}
```
On va sans doute faire évoluer ça, mais ça me parait plus clair comme URL (j'aurais du le faire pour les favoris aussi d'ailleurs)